frontend/src/metabase/actions/containers/ActionCreator/FormCreator/FormCreator.styled.tsx

```diff
@@ -83,3 +83,11 @@ export const TopRightIcon = styled(Icon)`
   top: 0;
   right: 0;
 `;
+
+export const WarningBanner = styled.div`
+  padding: ${space(2)};
+  border: 1px solid ${color("accent4")};
+  border-radius: ${space(1)};
+  background: rgba(249, 212, 92, 0.1);
```

Comment:
I have a few comments about this
1. `rgba` usage in the whole project is very minimal. I'm not even sure they're meant to be used directly.
2. there is `alpha` utility in `frontend/src/metabase/lib/colors/palette.ts` that we could use e.g.
    ```
    alpha("warning", 0.1)
    ```
    which will yield a very close color.

#### `alpha("warning", 0.1)`
![image](https://github.com/metabase/metabase/assets/1937582/2f797ca5-723d-42d7-987a-4003ccabff7b)

#### `alpha("accent4", 0.1)`
![image](https://github.com/metabase/metabase/assets/1937582/fe897680-cfca-463f-b552-1b1b30179d4e)


The reason we shouldn't use `accent4` is because that color could be changed in the appearance settings for pro and enterprise users 🫠
![image](https://github.com/metabase/metabase/assets/1937582/2b83e538-15ad-4fd4-835e-debb072bfc26)

@kdoh our [`accent4`](https://github.com/metabase/metabase/blob/6754ecf64272e2eebad70e377ddbf58a3b0c6278/frontend/src/metabase/lib/colors/palette.ts#L17) and [`warning`](https://github.com/metabase/metabase/blob/6754ecf64272e2eebad70e377ddbf58a3b0c6278/frontend/src/metabase/lib/colors/palette.ts#L27) are very close in color. Is that intentional? Judging from the figma file I guess we should replace `#F9CF48` with `#F9D45C` in `palette.ts`. What do you think? CC. @ranquild since you've worked extensively on colors.

Reply:
ah, I saw in codebase usage of rgba and didn't find alpha utils, TIL

Reply:
updated usage, thanks!